### PR TITLE
Add kubernetes event takeover and engage page

### LINKED ISSUE #7818: [Event] Kubernetes Cloud to Edge (US only)] - Build takeover and engage page
## Requirements

Please build a takeover and an engage page based on this [brief](https://docs.google.com/spreadsheets/d/18vsAl7oYtIJSSPuuMJUXipgUn9ZjveWt7F9B5rGS87o/edit#gid=0) and [design](2734):

Please note this is US only.
 
- [x] build takeover
- [x] build engage page
- [x] add takeover to `/takeovers/index.html`
- [x] add takeover to `/index.html`
- [x] add engage page to `/engage/index.html`
- [x] test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add takeover and engage page for US k8s event

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Reload the page until you see the "Kubernetes From Cloud to Edge" takeover
- Check that it matches [the brief](https://docs.google.com/spreadsheets/d/18vsAl7oYtIJSSPuuMJUXipgUn9ZjveWt7F9B5rGS87o/edit#gid=2113339190)
- Click through to the engage page and check that it matches [the brief](https://docs.google.com/spreadsheets/d/18vsAl7oYtIJSSPuuMJUXipgUn9ZjveWt7F9B5rGS87o/edit#gid=1271849439) and [the copy doc](https://docs.google.com/document/d/1LClShZkNAZYCHhXah452SEFYdwetyPstodwRL0Y9ZV8/edit)

## Issue / Card

Fixes #7818 

## Screenshots
screencapture-localhost-8001-engage-kubernetes-event-us-1592990056616

### CODE REVIEW

**Comment by steverydz** [comment]
@meralonso @spencerbygraves I've used the [Vanilla media object pattern](https://vanillaframework.io/docs/patterns/media-object) for the speaker images. Let me know if you think it should be something else.

File: templates/engage/kubernetes-event-us.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+31 July, 2020
+[2:30-4:30pm CT]
+
+**What it is**: A live, interactive virtual workshop with Canonical’s Kubernetes experts. This will include a presentation from our speakers, panel discussion incorporating live comments and questions from the audience, and free Kubernetes resources and demos to help you achieve your technology goals.
+
+**Who should attend**: Developers and enterprise IT teams
+wanting to learn more about how to overcome common Kubernetes challenges in a variety of contexts, and the most efficient solutions for executing their Kubernetes strategy.
+
+**What you will learn**: Best tools and strategies for streamlining your Kubernetes deployment and operations from public cloud, to containers, and even lightweight devices. We will also discuss the Kubernetes portfolio of Canonical and Ubuntu, and how its main components - MicroK8s and Charmed Kubernetes - can contribute to the flexibility you need deployment at every stage of your container DevOps cycle.
+
```
Comment (sowasred2012) [inline templates/engage/kubernetes-event-us.md:34]:
```suggestion
## Speakers:
```

Comment (sowasred2012) [reply templates/engage/kubernetes-event-us.md:34]: As it is you can easily infer these people are speakers, but thought this would make it explicit

**Comment by spencerbygraves** [comment]
@steverydz thanks, I think that's fine.

**Comment by meralonso** [comment]
@steverydz @spencerbygraves maybe something like this would be nicer?
Landing page - Kubernetes Cloud to Edge

**Comment by steverydz** [comment]
@meralonso the right column isn't possible with the current template

**Pull request merged**